tests/python/gaia-ui-tests/gaiatest/apps/gallery/app.py

```diff
@@ -21,6 +21,7 @@ class Gallery(Base):
     name = 'Gallery'
 
     _gallery_items_locator = (By.CSS_SELECTOR, 'div.thumbnail')
+    _overlay_locator = (By.ID, 'overlay')
```

Comment:
These python test changes seem good, but also seem unrelated to this patch. Did you mean for them to be here, or is this just a merge conflict sort of thing?